user-guide/Reference/faq/DataMiner_client_applications.md

```diff
@@ -20,7 +20,7 @@ In case of problems, Skyline Technical Support may ask you to provide version in
 
 ### Sending debug information to Technical Support
 
-1. In DataMiner Cube, go to *Apps* > *About*.
+1. In DataMiner Cube, go to *Apps* > *General section* > *About*.
```

Comment:
Only place something in italics if it is an exact reference to the UI. In this case, the UI says "General", not "General section". Also, because the user doesn't have to click "General" to get to "About", I'd prefer not to add this as an additional step. You can instead add it as extra info in parentheses:
1. In DataMiner Cube, go to *Apps* > *About* (in the *General* section).